docs/source/etl/merge.rst

```diff
@@ -0,0 +1,94 @@
+.. _merge-datasets:
+
+========================
+Merging Variant Datasets
+========================
+
+.. invisible-code-block:: python
+
+    import glow
+    glow.register(spark)
+    path1 = 'test-data/vcf-merge/HG00096.vcf.bgz'
+    path2 = 'test-data/vcf-merge/HG00097.vcf.bgz'
+
+Users frequently want to merge multiple VCF, BGEN, or Plink files from non-overlapping sample sets
```

Comment:
This wording is a little awkward. I think it'd be simpler to say "You can use Glow and Spark to merge genomic variants from non-overlapping sample sets into a multi-sample dataset."